src/components/Autocomplete/SectionItem/SectionItem.tsx

```diff
@@ -27,7 +27,7 @@ export default function SectionItem(props: SectionItemProps) {
 
   if (isProduct(item)) {
     defaultChildren = (
-      <>
+      <a href={item.data?.url}>
```

Comment:
small edge case, should we validate if there is no URL (not to render A element in that case)? clicking A with empty href will refresh the page